src/porepy/fracs/fractures.py

```diff
@@ -641,7 +641,7 @@ def __init__(self, fractures=None, domain=None, verbose=0, tol=1e-8):
 
         # Initialize with an empty domain. Can be modified later by a call to
         # 'impose_external_boundary()'
-        self.domain = domain
+        self.domain = pp.fracs.utils.domain_to_dict(domain)
```

Comment:
@IvarStefansson is right, documentation of options for domain is needed (also independent of this PR).